textflip/commandline.py

```diff
@@ -0,0 +1,15 @@
+import sys
+
+from .rotation import flip
+
+
+def flip_cli():
+    if len(sys.argv) > 1:
```

Comment:
One final thing, then I'm happy: I'd expext `flip foo bar bat` to print

> ʇɐq ɹɐq ooɟ

instead of just

> ooɟ

`print(flip(" ".join(sys.argv[1:])))` or `print(*(flip(word) for word in sys.argv[1:]))` should do the trick.